app/controllers/application_controller.rb

```diff
@@ -26,7 +27,8 @@ class ApplicationController < ActionController::Base
                 :login_anchor,
                 :comments_count,
                 :has_comments?,
-                :subject
+                :subject,
+                :ask_redirect?
```

Comment:
`ask_redirect` doesn't seem expressive enough to me. 

To which location is this redirection talking about?  Also remember that there are already  similar selector that are named `should_*?`. 

So I would name it something like `should_choose_organization?`  or something like that.